### ISSUE TITLE
A11y_Polyglot Notebooks Extension for VS Code_Table_ Name Role Value: Incorrect role is defined for "Share" buttons as link

### ISSUE THREAD

**Issue by kupatkar99** [dotnet/interactive#2464]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #DesktopApp; #Win32; #.NET Interactive; #WAG4.1.2; #Name Role Value; 
#SH-PolyglotNotebooksExtension-Win32-Nov2022;

### Environment and OS details:
Visual Studio Code - Insiders
Windows 11 Enterprise version 22H2 (OS Build 22623.875)

### Repro Steps:
1)Install "Visual Studio Code" insider application.
2)Install .NET 6.0 SDK using https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.402-windows-x64-installer.
3)Install Polyglot Notebook extension.
4)launch Visual studio code insider and Ctrl+Alt+Shift+N will invoke "Polyglot Notebook: Create new blank notebook" command used 
 above.
5)Enter any code and run the code.
6)Tab till values and press enter.
7)In Polyglotnotebooks:values section tab till "table".
8)Verify whether the correct role is defined for share button or not.

### Actual Result:
Incorrect role is defined for "Share" buttons as link.

### Expected Result:
Correct role should be defined for "Share" as buttons.
![A11y_ NETInteractive_Table_ Name Role Value](https://user-images.githubusercontent.com/98530069/201646461-941574e9-4ed9-4677-a582-b5c00e4b6184.png)

**Comment by VijayaBharathyA** [comment]
#Closed
**Verified the issue in Below Environment:**
VS code Version: 1.74.0-insider (user setup)
Windows 11 Pro Version	22H2
Windows Feature Experience Pack 1000.22636.1000.0
Issue is fixed. Hence Closing bug

![Correct role should be defined ](https://user-images.githubusercontent.com/100189407/205565872-f7d8315b-ef90-4f59-a3d8-cda8bd62e5cd.jpg)
